src/archiefvernietigingscomponent/destruction/utils.py

```diff
@@ -0,0 +1,118 @@
+from datetime import date, datetime
+
+from django.shortcuts import render
+from django.utils.translation import gettext as _
+
+from zds_client import ClientError
+
+from archiefvernietigingscomponent.destruction.constants import (
+    ListItemStatus,
+    ReviewStatus,
+)
+from archiefvernietigingscomponent.destruction.models import (
+    DestructionList,
+    DestructionListReview,
+)
+from archiefvernietigingscomponent.destruction.service import (
+    fetch_process_type,
+    fetch_zaaktype,
+)
+
+
+class NoClientException(Exception):
+    """No client could be built"""
+
+    pass
+
+
+def get_looptijd(zaak: dict) -> int:
+
+    if zaak["einddatum"] != "":
+        end_date = datetime.strptime(zaak["einddatum"], "%Y-%m-%d")
+    else:
+        end_date = date.today()
+
+    return (end_date - datetime.strptime(zaak["startdatum"], "%Y-%m-%d")).days
+
+
+def get_vernietigings_categorie_selectielijst(zaaktype: str) -> str:
+    try:
+        zaaktype = fetch_zaaktype(zaaktype)
+    except ClientError as exc:
+        # If the zaaktype couldn't be retrieved, return an empty value
+        return ""
+
+    if not zaaktype.get("selectielijstProcestype"):
+        return ""
+
+    try:
+        process_type = fetch_process_type(zaaktype["selectielijstProcestype"])
+    except ClientError as exc:
+        # If the process type couldn't be retrieved, return an empty value
+        return ""
+
+    return str(process_type["nummer"])
+
+
+def get_destruction_list_archivaris_comments(destruction_list: DestructionList) -> str:
+    review = (
+        DestructionListReview.objects.filter(
+            destruction_list=destruction_list,
+            status=ReviewStatus.approved,
+            author__role__can_review_destruction=True,
+            author__role__can_view_case_details=False,
```

Comment:
the same question as before - are we sure, that all these filters are needed here?